README.md

```diff
@@ -14,3 +14,47 @@ provided by the bot. You will only need to do this once across all repos using o
 This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/).
 For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or
 contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
+
+## Join the Community hub to contribute content
+
+1. Go to the **hub** node in the **Community** workspace.
```

Comment:
Go to the Community hub node in the Community workspace.

Reply:
Added in commit https://github.com/microsoft/configmgr-hub/pull/11/commits/0ebfdd1ccb14303527e04f2ac12bca41c1d16c77